external/derby/build.xml

```diff
@@ -21,9 +31,30 @@
 		</exec>
 	</target>
 
-	<target name="build_image" depends="clean_image" description="build derby test docker image">
+	<target name="move_scripts" depends="clean_image" description="">
+	<echo message=" external/derby move scripts ........ ${top}"/>
+		<copy todir="${EXT_DEST}">
+			<fileset dir="${top}" includes="*.sh"/>
+		</copy>
+	</target>
+
+	<target name="build_dockerfile" depends="move_scripts" description="build derby test dockerfile">
+		<echo message="Generate dockerfile!!! : build_image.sh ${TEST} ${VERSION} ${VM} ${OS} ${PACKAGE} ${BUILD}" />						
+		<exec executable="bash">
+			<arg value="${EXT_DEST}/build_image.sh"/>
+			<arg value="${TEST}"/>
+			<arg value="${VERSION}"/>
+			<arg value="${VM}"/>
+			<arg value="${OS}"/>
+			<arg value="${PACKAGE}"/>
+			<arg value="${BUILD}"/>
+		</exec>
+	</target>
+
+	<target name="build_image" depends="build_dockerfile" description="build derby test docker image">
```

Comment:
You can take this step completely out. The `build_image` script should generate the image and build said generated Dockerfile.